src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/validation/SimpleSwitchRule.java

```diff
@@ -37,7 +37,7 @@ public class SimpleSwitchRule {
     private SwitchId switchId;
     private long cookie;
     private int inPort;
-    private int outPort;
+    private int outPort; //TODO:5025  change this field to PortNumber type if possible.
```

Comment:
5025 meaning the issue for tracking? But this is the current issue which will be closed when this PR is completed. Could you create a separate issue for this todo?

Reply:
Created a separate task for it